src/components/DocumentationTopic/DocumentationNav.vue

```diff
@@ -140,6 +146,10 @@ export default {
       type: String,
       required: false,
     },
+    versionList: {
+      type: Array,
+      required: false,
```

Comment:
Default version is not required so no need to specify.
Also add a default value to it or you will receive null and it will be in conflict with the type of Array

```suggestion
      default: () => [],
```